app/utils/answersHelpers/consignment/houseConsignment/ConsignmentItemAnswersHelper.scala

```diff
@@ -99,6 +100,23 @@ class ConsignmentItemAnswersHelper(
         )
     }
 
+  def additionalInformationsSection: Section = {
```

Comment:
I feel like this should be a Seq[Section] ? Use additional references as an example

Reply:
Refactored...